test/test_distributions.py

```diff
@@ -1165,6 +1171,7 @@ def inv_vec_transform(y):
 @pytest.mark.parametrize('transform, event_shape', [
     (PermuteTransform(jnp.array([3, 0, 4, 1, 2])), (5,)),
     (PowerTransform(2.), ()),
+    (SoftplusTransform, ()),
```

Comment:
Should be `SoftplusTransform()` I guess

Reply:
Yes, this worked! I can add more complex `codomain` stuff now if that would be useful.

Reply:
I think this is fine. :)